Fixed indentation error in `facts-demo.yml` example.

onenetworkguy opened the pull request:
##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->

<!---
If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

`content` and `dest` should be indented under the `copy` module in [Network Best Practices User Guide 2.5](http://docs.ansible.com/ansible/devel/network/user_guide/network_best_practices_2.5.html)

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Docs Pull Request

##### COMPONENT NAME
<!--- Name of the module, plugin, module or task -->
N/A Docs Pull Request
##### ANSIBLE VERSION
<!--- Paste verbatim output from "ansible --version" between quotes below -->
N/A Docs Pull Request

##### ADDITIONAL INFORMATION
<!---
Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful.
  -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
N/A Docs Pull Request

Commit:
Fixed indentation error in `facts-demo.yml` example.

`content` and `dest` should be indented under the `copy` module.

gundalow reviewed (approved):
Thanks for this. My fault I messed up the formatting when I copied the sample in.

FYI we use inline samples for new bits so this shouldn't happen https://github.com/ansible/ansible/tree/devel/docs/docsite/rst/network/getting_started/sample_files

Commit:
Updated indentation of copy module in `facts-demo.yml` example.

Updated indentation of copy module in `facts-demo.yml` to be inline with its name.
